pymobiledevice3/services/accessibilityaudit.py

```diff
@@ -100,16 +101,30 @@ def issue_type(self) -> typing.Any:
     def ml_generated_description(self) -> typing.Any:
         return self._fields['MLGeneratedDescriptionValue_v1']
 
+    @property
+    def long_description_extra_info(self) -> typing.Any:
+        return self._fields['ElementLongDescExtraInfo_v1']
+
     @property
     def font_size(self) -> typing.Any:
         return self._fields['FontSizeValue_v1']
 
+    @property
+    def foreground_color(self) -> typing.Any:
+        return self._fields['ForegroundColorValue_v1']
+
+    @property
+    def background_color(self) -> typing.Any:
+        return self._fields['BackgroundColorValue_v1']
+
     def json(self) -> typing.Mapping:
-        resp = {}
-        resp['element_rect_value'] = self.rect
-        resp['issue_classification'] = self.issue_type
-        resp['font_size'] = self.font_size
-        resp['ml_generated_description'] = self.ml_generated_description
+        resp = {'element_rect_value': self.rect, 'issue_classification': self.issue_type, 'font_size': self.font_size,
+                'ml_generated_description': self.ml_generated_description,
+                'long_description_extra_info': self.long_description_extra_info}
+        # Include foreground and background colors when issue type is 'testTypeContrast'
+        if self._fields['IssueClassificationValue_v1'] == 12 or self._fields['IssueClassificationValue_v1'] == 13:
```

Comment:
- Use `IntEnum` if possible
- Use `if self._fields['IssueClassificationValue_v1'] in (NewIntEnum.A, NewIntEnum.b)` instead